core/src/smartcontracts/wasm.rs

```diff
@@ -177,9 +181,16 @@ struct State<'wrld> {
 
 impl<'wrld> State<'wrld> {
     fn new(wsv: &'wrld WorldStateView, account_id: AccountId, config: Configuration) -> Self {
+        #[allow(clippy::expect_used)]
+        let unix_timestamp_millis = current_time()
+            .as_millis()
+            .try_into()
+            .expect("Timestamp millis should fit into u64");
```

Comment:
```suggestion
            .unwrap_or(0);
```
The odds of this happening naturally are 0. Plus we get a signal that the timestamp for one reason or another didn't fit.